gradle/dependency-management/capabilities.json

```diff
@@ -86,5 +86,13 @@
         ],
         "selected": "org.ow2.asm:asm-util",
         "upgrade": "6.1"
+    },
```

Comment:
@melix now I am trying to use `capabilities.json`, would you like to take a look at it?

Reply:
@oehme As suggested by @melix , now I removed the redundant usage of constraints. Hopefully this can make our code base clearer.